pages/products/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useRef, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+
+import { client } from "../../.tina/__generated__/client";
+import { useTina } from "tinacms/dist/react";
+
+import { Layout } from "../../components/layout";
+import { Container } from "../../components/util/container";
+import { Breadcrumbs } from "../../components/blocks/breadcrumbs";
+import { SEO } from "../../components/util/seo";
+import { InferGetStaticPropsType } from "next";
+
+export default function ProductsIndex(
+  props: InferGetStaticPropsType<typeof getStaticProps>
+) {
+  const gridRef = useRef(null);
+  const { data } = useTina({
+    query: props.query,
+    variables: props.variables,
+    data: props.data,
+  });
+
+  const [productsList, setProductsList] = useState([]);
+  const [seo, setSeo] = useState(null);
+
+  useEffect(() => {
+    // extract the data we need from the tina result
+    const processedData = processData(data);
+    setProductsList(processedData.productsList);
+    setSeo(processedData.seo);
+  }, [data]);
+
+  return (
+    <Layout>
+      <SEO seo={{ ...seo, canonical: "/products" }} />
```

Comment:
Can we ge the `canonical` from the seo object?

```suggestion
       <SEO seo={seo} />
```

Reply:
> Can we get the `canonical` from the seo object?

Yes, we can, done. 😊